package Bitmeyen_Exceptionlar;

public class Normal_Exception {

    public static void main(String[] args)  {


        int a = 7;
        int b = 0;


         // throwslar run time exception i handl etmede bir ise yaramiyor



    }//main
}
